server/log_operation_manager_test.go

```diff
@@ -171,15 +172,58 @@ func TestLogOperationManagerPassesIDs(t *testing.T) {
 
 	mockLogOp := NewMockLogOperation(ctrl)
 	infoMatcher := logOpInfoMatcher{50}
-	mockLogOp.EXPECT().ExecutePass(gomock.Any(), logID1, infoMatcher)
-	mockLogOp.EXPECT().ExecutePass(gomock.Any(), logID2, infoMatcher)
+	mockLogOp.EXPECT().ExecutePass(gomock.Any(), logID1, infoMatcher).Return(1, nil)
+	mockLogOp.EXPECT().ExecutePass(gomock.Any(), logID2, infoMatcher).Return(0, nil)
 
 	info := defaultLogOperationInfo(registry)
 	lom := NewLogOperationManager(info, mockLogOp)
 
 	lom.OperationSingle(ctx)
 }
 
+func TestLogOperationManagerExecutePassError(t *testing.T) {
+	ctx := context.Background()
+	logID1 := int64(451)
+	logID2 := int64(145)
+	logID1Label := strconv.FormatInt(logID1, 10)
+	logID2Label := strconv.FormatInt(logID2, 10)
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	fakeStorage, mockAdmin := setupLogIDs(ctrl, map[int64]string{451: "LogID1", 145: "LogID2"})
+	registry := extension.Registry{
+		LogStorage:   fakeStorage,
+		AdminStorage: mockAdmin,
+	}
+
+	mockLogOp := NewMockLogOperation(ctrl)
+	infoMatcher := logOpInfoMatcher{50}
+	mockLogOp.EXPECT().ExecutePass(gomock.Any(), logID1, infoMatcher).Return(1, nil)
+	mockLogOp.EXPECT().ExecutePass(gomock.Any(), logID2, infoMatcher).Return(0, errors.New("test error"))
+
+	signingRunsSnapshot := testonly.SnapshotCounter(signingRuns, logID1Label, logID2Label)
```

Comment:
... consequently, these calls will need to pass 2 slices of strings instead of 2 strings.